languages/ru-RU.js

```diff
@@ -0,0 +1,196 @@
+const { Language, util } = require('klasa');
+
+module.exports = class extends Language {
+
+	constructor(...args) {
+		super(...args);
+		this.language = {
+			DEFAULT: (key) => `Ключ ${key} ещё не переведён на ru-RU.`,
+			DEFAULT_LANGUAGE: 'Язык по-умолчанию',
```

Comment:
A hyphen is not needed there, so it should be "по умолчанию".
```suggestion
			DEFAULT_LANGUAGE: 'Язык по умолчанию',
```